node/declaration.ts

```diff
@@ -30,19 +30,44 @@ export const getDeclarationsFromConfig = (
   // a function configuration object, we replace the path because that object
   // takes precedence.
   for (const declaration of [...tomlDeclarations, ...deployConfig.declarations]) {
-    const config = functionsConfig[declaration.function] ?? {}
+    const config = functionsConfig[declaration.function]
+
+    // If no config is found, add the declaration as is
+    if (!config) {
+      declarations.push(declaration)
+
+      // If we have path specified create a declaration for each path
+    } else if (config.path?.length) {
+      // eslint-disable-next-line max-depth
+      if (!Array.isArray(config.path)) config.path = [config.path]
```

Comment:
Rather than mutating config here, I think it would be nicer to create a new variable with the normalised values. Something like:

```js
const paths = Array.isArray(config.path) ? config.path : [config.path]
```

I _think_  that should also let you get rid of the `eslint-disable-next-line` directive.

Reply:
Nice, I wanted to work on getting rid of the mutations yesterday but my brain was a bit foggy, I like this change a lot

Reply:
Are you still planning on doing this?

Reply:
:D @eduardoboucas guess who forgot to push her changes, it's me! 

Should be up to date now